applications/nrf5340_audio/doc/firmware_architecture.rst

```diff
@@ -7,39 +7,40 @@ nRF5340 Audio overview and firmware architecture
+All the applications use the same code base, except different :file:`main.c` files.
 
-Regardless of the configuration, the application handles the audio data in the following manner:
+You might need to configure and program two applications for testing the interoperability, depending on your use case.
+See :ref:`nrf53_audio_app_tutorial_cis` and :ref:`nrf53_audio_app_tutorial_bis` for more information.
 
-1. The gateway receives audio data from the audio source over USB or I2S.
-#. The gateway processes the audio data in its application core, which channels the data through the application layers:
+.. _nrf53_audio_app_overview_differences:
 
-   a. Audio data is sent to the synchronization module (I2S-based firmware) or directly to the software codec (USB-based firmware).
-   #. Audio data is encoded by the software codec.
-   #. Encoded audio data is sent to the Bluetooth LE Host.
+Differences between apps
+************************
 
-#. The host sends the encoded audio data to the LE Audio Controller Subsystem for nRF53 on the network core.
-#. The subsystem forwards the audio data to the hardware radio and sends it to the headset devices, as per the LE Audio specifications.
-#. The headsets receive the encoded audio data on their hardware radio on the network core side.
-#. The LE Audio Controller Subsystem for nRF53 running on each of the headsets sends the encoded audio data to the Bluetooth LE Host on the headsets' application core.
-#. The headsets process the audio data in their application cores, which channel the data through the application layers:
+The following table summarizes the differences between the available nRF5340 Audio applications.
 
-   a. Audio data is sent to the stream control module and placed in a FIFO buffer.
-   #. Audio data is sent from the FIFO buffer to the synchronization module (headsets only use I2S-based firmware).
-   #. Audio data is decoded by the software codec.
+.. list-table:: Differences between nRF5340 Audio applications
+   :header-rows: 1
 
-#. Decoded audio data is sent to the hardware audio output over I2S.
+   * - Application
+     - LE Audio mode and role
+   * - :ref:`Unicast client<nrf53_audio_unicast_client_app>`
+     - CIS gateway
+   * - :ref:`Unicast server<nrf53_audio_unicast_server_app>`
+     - CIS headset
+   * - :ref:`Broadcast sink<nrf53_audio_broadcast_sink_app>`
+     - BIS gateway
+   * - :ref:`Broadcast source<nrf53_audio_broadcast_source_app>`
+     - BIS headset
 
-In the `I2S-based firmware for gateway and headsets`_, sending the audio data through the application layers includes a mandatory synchronization step using the synchronization module.
+If you :ref:`select the I2S serial connection<nrf53_audio_app_configuration_select_i2s>` to enable the `I2S-based firmware for gateway and headsets`_, sending the audio data through the application layers includes a mandatory synchronization step using the synchronization module.
```

Comment:
Even though USB is used, we still need to have this sync module in place on the headset side.

Reply:
Meaning the file is there also when we use USB?

Reply:
Removed the line during the meeting.